WooCommerce/Classes/ViewRelated/Orders/OrdersViewController.swift

```diff
@@ -260,6 +274,15 @@ extension OrdersViewController {
     @objc func defaultAccountWasUpdated() {
         syncingCoordinator.resetInternalState()
     }
+
+    /// Default Site Updated Handler
+    ///
+    @objc func defaultSiteWasUpdated() {
+        // dump old orders cache
+        syncingCoordinator.resynchronize()
+        // reload the results controller
+        reloadResultsController()
+    }
```

Comment:
I listen for the notification `StoresManagerDidUpdateDefaultSite` and that triggers this method. But my comments (dump old orders cache) do not match what is happening.